package-res/resources/web/pentaho/visual/ccc/_axes/AbstractAxis.js

```diff
@@ -21,7 +21,7 @@ define([
   "use strict";
 
   return Base.extend({
-    constructor: function(chart, axisId, gems) {
+    constructor: function(chart, axisId, mais) {
       this.chart = chart;
```

Comment:
Why renaming `gem`/`gems` to  `mai`/`mais` and not the slightly longer (but more comprehensible) `mapAttrInfo`/`mapAttrInfos` ? 

I find that the shorter variant introduces a higher effort to parse and a higher brain utilization. The benefit of a short variable/method name is lost if you need to read the code around to figure out wtf is `mai`